test_scripts/Capabilities/PersistingHMICapabilities/common.lua

```diff
@@ -556,19 +586,43 @@ function m.registerAppsSuspend( pCapResponse, pHMIParams )
   local mobSession2 = actions.mobile.getSession(appSessionId2)
   local mobSession3 = actions.mobile.getSession(appSessionId3)
 
-  actions.hmi.getConnection():ExpectNotification("BasicCommunication.OnAppRegistered",
-    { application = { appName = actions.app.getParams(appSessionId1).appName }},
-    { application = { appName = actions.app.getParams(appSessionId2).appName }},
-    { application = { appName = actions.app.getParams(appSessionId3).appName }})
-  :Do(function(exp, data)
-      if exp.occurences == 1 then
-        actions.app.setHMIId(data.params.application.appID, appSessionId1)
-      elseif exp.occurences == 2 then
-        actions.app.setHMIId(data.params.application.appID, appSessionId2)
-      else
-        actions.app.setHMIId(data.params.application.appID, appSessionId3)
-      end
-    end)
+  local isAppSessionId = {
```

Comment:
@VjKlepikov The name of the variable is not reflect its purpose. I would propose to rename it
```suggestion
  local isAppRegisteredReceived = {
```

Reply:
Fixed (see 3ab319)